controller/work_item_link.go

```diff
@@ -513,3 +518,14 @@ func ConvertLinkToModel(appLink app.WorkItemLinkSingle) (*link.WorkItemLink, err
 
 	return &modelLink, nil
 }
+
+func ValidateSingleParent(ctx *workItemLinkContext, targetID uint64, linkTypeID uuid.UUID) error {
+	result := link.WorkItemLink{}
+	var db *gorm.DB
+	db = db.Where("link_type_id=? AND target_id=?", linkTypeID, targetID).Find(&result)
```

Comment:
@baijum no, I am not adding any db constraint. I am going to add restriction in code.